source/common/ssl/context_impl.cc

```diff
@@ -253,12 +253,13 @@ bssl::UniquePtr<SSL> ContextImpl::newSsl() const {
 int ContextImpl::verifyIgnoreCertExpirationCallback(int ok, X509_STORE_CTX* ctx) {
   int err = X509_STORE_CTX_get_error(ctx);
```

Comment:
This should be inside `(!ok)` block, since that's an extra call that's unused outside of it.